### PR TITLE
[DOCS-5456] AWS AI connectors

### PR DESCRIPTION
**Pull request by m-hulbert**

### CODE REVIEW

File: docs/modeling/connectors/ootb/comprehend.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+The Comprehend connector requires an AWS account to access Amazon features. It also requires an [Identity and Access Management (IAM)](https://aws.amazon.com/iam/) user to have the `textract:DetectDocumentText` and `textract:AnalyzeDocument` permissions. 
+
+## Input parameters
+The following are the parameters that can be passed to the Comprehend connector as input parameters using the `ENTITY` action:
+
+| Parameter | Description | Type | Required? |
+| --------- | ----------- | ---- | --------- | 
+| `nodeId` | The node ID of the file to use from Alfresco Content Services | String | `*` |
+| `uri` | The URI of the file to use | String | `*` |
+| `files` | A [file](../../modeling/files.md) uploaded in a process and set as a process variable | File | `*` |
```
Comment (mmartinadanx) [inline docs/modeling/connectors/ootb/comprehend.md]: Just to clarify. Files can be uploaded in a process (process files), but can also be uploaded in a form (by the end user) or produced by other connectors (e.g. Aspose connector)

**Review by mmartinadanx — COMMENTED** [review]
just a clarification

File: docs/modeling/connectors/ootb/textract.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+| `uri` | The URI of the image to use | String | `*` |
+| `files` | An image [file](../../modeling/files.md) uploaded in a process and set as a process variable | File | `*` |
+| `outputFormat` | Sets the output format to `JSON` or `txt`. The default is `JSON` | String | No |
+| `confidenceLevel` | The minimum confidence level to use for a label. The default is 0.75 | String | No |
+| `timeout` | The timeout period for calling the Textract service in milliseconds | Integer | No | 
+
+`*` Only one of these parameters is required.
+
+## Output parameters
+The following are the parameters that are returned to the process by the Textract connector as output parameters using the `GENERATE` action:
```
Comment (mmartinadanx) [inline docs/modeling/connectors/ootb/textract.md]:
```suggestion
The following are the parameters that are returned to the process by the Textract connector as output parameters using the `EXTRACT` action:
```

**Review by mmartinadanx — CHANGES_REQUESTED** [review]
One minor fix

File: docs/modeling/connectors/ootb/comprehend.md (hunk 1 above)
Comment (m-hulbert) [reply docs/modeling/connectors/ootb/comprehend.md]: Thanks for the clarification - I've updated all 3 connectors to note about the additional sources

**Review by mmartinadanx — APPROVED** [review]

**Pull request merged**